### PR TITLE
Share ios user data docs for android and web

### PR DESCRIPTION
**Pull request by alavers**
@mspensieri @jpjoyal @jugarrit @lemieux

### CODE REVIEW

File: source/includes/users/_user_data_intro.md [hunk 1]
```diff
@@ -1,3 +1,96 @@
...
+
+```objective_c
+[SKTUser currentUser].firstName = @"Doctor";
+[SKTUser currentUser].lastName = @"Who";
+```
+```swift
+SKTUser.currentUser().firstName = "Doctor"
+SKTUser.currentUser().lastName = "Who"
+```
+```java
```
Comment (mspensieri) [inline source/includes/users/_user_data_intro.md]: no `javascript` examples?

```
Smooch.updateUser({
    givenName: 'xyz',
    surname: 'abc'
});
```

File: source/includes/users/_user_data_intro.md [hunk 2]
```diff
@@ -1,3 +1,96 @@
 ## User data
 
-Smooch allows you to specify profile information about your users so that you can better understand the context around a user's message. 
\ No newline at end of file
+Smooch allows you to specify profile information about your users so that you can better understand the context around a user's message.
+
+```java
```
Comment (mspensieri) [inline source/includes/users/_user_data_intro.md]: same question, no iOS examples here?

File: source/includes/users/_user_data_intro.md (hunk 1 above)
Comment (alavers) [inline source/includes/users/_user_data_intro.md]: Good point, will add

File: source/includes/users/_user_data_intro.md (hunk 2 above)
Comment (alavers) [inline source/includes/users/_user_data_intro.md]: The iOS examples are interspersed below, including them all here would be repetitive.

I repeated it for Android, but I don't think its necessary. I'll make some changes.

**Pull request merged**